c2cgeoform/static/src/styles.js

```diff
@@ -26,13 +26,18 @@ export function getDefaultStyle() {
 }
 
 export function getStyleFunction(options) {
-  return feature =>
-    new Style({
-      image: new Icon({
-        anchorXUnits: 'fraction',
-        anchorYUnits: 'pixels',
-        src: feature.get('icon') || options.icon || defaultIconUrl,
-        opacity: feature == options.context?.feature ? 1 : options.opacity,
-      }),
-    })
+  return feature => {
```

Comment:
This style function generates lots of garbage. Styles should be cached & reused.